tests/unit/test_utils/test_vdb/test_elastic_vdb.py

```
@@ -481,18 +461,16 @@ def test_delete_collections(
             f"Collections deleted: {collection_names}"
         )
 
-    @patch("nvidia_rag.utils.common.get_config")
     @patch("nvidia_rag.utils.vdb.elasticsearch.elastic_vdb.Elasticsearch")
     @patch("nvidia_rag.utils.vdb.elasticsearch.elastic_vdb.VectorStore")
     @patch("nvidia_rag.utils.vdb.elasticsearch.elastic_vdb.get_unique_sources_query")
     def test_get_documents(
-        self, mock_sources_query, mock_vector_store, mock_elasticsearch, mock_get_config
-    ):
+        self, mock_sources_query, mock_vector_store, mock_elasticsearch):
```

Ruff (0.14.5)

Unused method argument: `mock_vector_store`

(ARG002)